dynamo-api/src/main/java/org/dynamoframework/configuration/DefaultProperties.java

```diff
@@ -0,0 +1,46 @@
+package org.dynamoframework.configuration;
+
+public interface DefaultProperties {
+    org.dynamoframework.domain.model.AttributeBooleanFieldMode getBooleanFieldMode();
```

Comment:
Is er een goede reden dat dit allemaal de fully qualified names zijn?

Reply:
Nee, die is er niet. Ik pas dit aan.